packages/components/src/buttons/src/Button.module.css

```diff
@@ -263,6 +377,16 @@
     user-select: none;
 }
 
+.hop-Button--ghost-primary[data-hovered]:not([data-loading]) .hop-Button__text,
```

Comment:
When the ghost button is focus-visible, it should have the same background as when it's hovering. `--hop-neutral-surface-hover`

<img width="615" alt="Capture d’écran, le 2024-03-28 à 15 16 01" src="https://github.com/gsoft-inc/wl-hopper/assets/2379339/5aae7711-5ce4-41d9-bca7-6f878c9b9b19">

Reply:
It is actually missing for all buttons. When a button is in :focus-visible, it must have the same background color, color and border-color (so basically all styling) as the hover state.